README.md

```diff
@@ -384,6 +384,24 @@ on the wiki.
 
 Bats is [SemVer compliant](https://semver.org/).
 
+*0.5.0* (June xx, 2018)
+
+`0.5.0` generally preserves compatibility with `0.4.0`, but with some Bash compatibility improvements and a massive performance boost. In other words:
+- all existing tests should remain compatible
+- tests that might've failed or exhibited unexpected behavior on earlier versions of Bash should now also pass or behave as expected
+
+Changes:
+
+* Added support for Docker.
+* Added support for test scripts that have the [unofficial strict mode](http://redsymbol.net/articles/unofficial-bash-strict-mode/) enabled.
+* Improved stability on Windows and macOS platforms.
+* Massive performance improvements, especially on Windows (#8)
+* Workarounds for inconsistent behavior between Bash versions (#82)
+* Workaround for preserving stack info after calling an exported function under Bash < 4.4 (#87)
+* Fixed TAP compliance for skipped tests
+* Added support for tabs in test names.
+* Fixed an installation bug where if Bats was uploaded from Windows to a server installation would fail (#88)
```

Comment:
Actually, that would be #91 now. Also it seems it could be worded better, something like: `bin/bats` and `install.sh` now work reliably on Windows (#91).

Reply:
I meant that the new line should replace this one, but no matter; I'll delete the old line during the cleanup pass.